### DISCUSSION TITLE
Review rel attribute values for tag <a>

### DISCUSSION THREAD

**Discussion by y4izus** [Qiskit/qiskit.org#2380] [Ideas]
In tag `<a>` there are three attribute values for `rel` that we have to have in mind:
- `noopener`: use it on all links with target _blank to avoid a malicious attack coming from the opened tab. _**We are already using it**_
- `noreferrer`: use it as similar as `noopener`, especially for older browsers. It makes sense to use them both. It will change the referred page from "coming from X" to "coming from direct search" (keep in mind if it's used for internal links)
 - `nofollow`: use it for SEO purposes to not to pass the link juice/PageRank to the linked page

We should have a discussion on this two points:
- We should ask if from the social accounts they want to know from where the traffic is coming. In this case, we should remove the `noreferrer` attribute (with it, the traffic will see as "they come from direct search" which is not the case). @delapuente, @lerongil , @CatherineKlauss do you know who manage the statistics of the social accounts?
- Reasons to use `nofollow` [here](https://www.searchenginejournal.com/use-nofollow-links). We don't have links only in the menu bar and in the footer, we also have on the experiments pages and on the events pages. We should analyze this links to take a decision about the `nofollow` attribute (notice that this links are generated from a markdown file, so there are more thing involve in this decision)

**Comment by delapuente** [comment]
This requires additional input from product. I'm removing it from the sprint backlog.

**Comment by lerongil** [comment]
Per discussion: 

The noreferrer attribute should be not be present outside of the qiskit ecoystem: 

qiskit.org/ 
youtube.com/qiskit
medium.com/qiskit
twitter.com/qiskit

**Comment by lerongil** [comment]
Split up nofollow to a separate issue
